iam.tf

```diff
@@ -37,3 +37,69 @@ resource "aws_iam_role_policy_attachment" "route53" {
   policy_arn = aws_iam_policy.route53[0].arn
   role       = lookup(module.eks.info.nodes.roles[count.index], "name")
 }
+
+data "aws_iam_policy_document" "create_eks_role" {
+  statement {
+    sid = "EKSDeployerEKS"
+    actions = [
+      "eks:*"
```

Comment:
Given that we are close to `*` https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelastickubernetesservice.html .